docs/papers/squad-paper.md

```diff
@@ -60,14 +108,24 @@ adopter, and more (detailed below). Each contribution is associated
 with a bonding curve whose token can be given value by the software
 developers (detailed below).
 
+The key point here is that **the original authors & early adopters get rewarded
+if the contribution becomes popular, creating a strong incentive to create and 
+discover potentially popular contributions.**
+
+Curved bonds also allow speculation, which has some positives and negatives,
+but probably overall positives (provide liquidity, better signalling on average).
+
 #### Right of first purchase
 
 When a contribution is made, the contributor may purchase as many of
-the definition tokens as they wish. Sinse this will always be the
+the definition tokens as they wish. Since this will always be the
 first purchase of that bond, it cannot fall in value but can increase
-in value. <<<<<Consider the option that contributors always get the first
+in value. 
+<<<<<Consider the option that contributors always get the first
 N tokens of their contributions (for free, or some fixed small fee on
-top of the ETH fee?, is spam even a problem?) and that funding the Gov
+top of the ETH fee?, is spam even a problem? 
+Ez: I don't think spam is a problem here: pump and dump / frontrunning are more likely, I think.
```

Comment:
Yeah probably true, though that's a problem no matter how we structure the discount